sql/tpch/queries/q14.sql

```diff
@@ -0,0 +1,9 @@
+SELECT 100.00 * SUM(CASE
+                        WHEN p_type LIKE 'PROMO%' THEN l_extendedprice * (1 - l_discount)
+                        ELSE 0
+    END) / SUM(l_extendedprice * (1 - l_discount)) AS promo_revenue
+FROM lineitem,
+     part
+WHERE l_partkey = p_partkey
+  AND l_shipdate >= DATE '1995-09-01'
+  AND l_shipdate < DATE '1995-09-01';
```

Comment:
` + interval '1' month` is missing.

Should be:
`AND l_shipdate < DATE '1995-09-01' + INTERVAL '1' month`